server/src/main/java/org/elasticsearch/cluster/coordination/ClusterFormationFailureHelper.java

```diff
@@ -157,7 +157,6 @@ String getDescription() {
                 if (INITIAL_MASTER_NODES_SETTING.get(Settings.EMPTY).equals(INITIAL_MASTER_NODES_SETTING.get(settings))) {
                     bootstrappingDescription = "[" + INITIAL_MASTER_NODES_SETTING.getKey() + "] is empty on this node";
                 } else {
-                    // TODO update this when we can bootstrap on only a quorum of the initial nodes
```

Comment:
Done, see call to `ClusterBootstrapService::isBootstrapPlaceholder` in `describeQuorum`.